scripts/us_eia/eia_860/utils.py

```diff
@@ -48,7 +48,7 @@ def state_alpha2_to_dcid(alpha2: object) -> str:
 
 def zip_to_dcid(zip: object) -> str:
     if pd_types.is_number(zip):
-        return f'dcid:zip/{zip:0>5}'
+        return f'dcid:zip/{zip:0>5.0f}'
```

Comment:
For future reference, could you note what changed with 3.9 that necessitates these changes?

Reply:
tbh i did not isolate these changes to python / pandas. i'll add a note to the PR though.